deploy/kubernetes/manifests/23-shipping-dep.yaml

```diff
@@ -20,6 +20,8 @@ spec:
       - name: shipping
         image: weaveworksdemos/shipping:0.4.8
         env:
+         - name: ZIPKIN
+           value: zipkin.jaeger.svc.cluster.local
```

Comment:
do we still need this?

Reply:
I don't know if it's necessary for jaeger, should take a look